doc/manual/tutorial/examples.md

```diff
@@ -222,6 +222,28 @@ task.submitAndWait(taskParameters).then((results) => {
     // Display error.
   });
 ```
+## List and Retrieve Workspace Files
+The GSF [**Job**] object provides the ability to fetch a list of workspace files.  It is required that the 'enableNodeInfo' configuration is set to true on the server for job workspace access.
+
+The task below, CustomTask, is a fictional task that writes a text file to the job workspace.  This also provides an example of using async/await instead of promises.
+```javascript
+const task = GSF.client({address:'MyServer',port:'9191'}).service('ENVI').task('CustomTask');
+
+// Submit job.
+const job = await task.submit({});
+
+// Wait for job to complete.
+await job.wait();
+
+// Get the list of workspace files.
+const workspaceFiles = await job.workspace();
```

Comment:
Maybe we can show an example here of how to loop through and console.log the list so users get a sense of what the return value is? E.g. is it just an array of strings or something else? Not required at all, just a thought.

Reply:
Sounds great @gsf-sellis! I have made the changes, let me know what you think.